src/components/HelpScoutBeacon/useBeaconSuggestions.js

```diff
@@ -0,0 +1,106 @@
+import { useEffect, useCallback } from 'react'
+
+const aboutAragon = '5c98a9330428633d2cf3fd23'
+
+const sectionToSuggestions = new Map([
+  [
+    'home',
+    [
+      '5cce35302c7d3a177d6e5b78', // home
```

Comment:
I'm guessing there's no good way to name things to a more friendly identifier on HelpScout?

Reply:
Yeah, there isn't

Reply:
Nope, it is horrible, at first glance I though the url links could be used (`article/16-home`) which weren't that bad but no :(